Tests/TensorFlowTests/OperatorTests/ComparisonTests.swift

```diff
@@ -28,8 +28,22 @@ final class ComparisonOperatorTests: XCTestCase {
         XCTAssertTrue(x < y)
     }
 
+    func testIsAlmostEqual() {
```

Comment:
Could you please add some tests involving approximate self equality of nan/infinity?
`Tensor<Float>([.nan, .infinity])`